LibreHardwareMonitorLib/LibreHardwareMonitorLib.csproj

```diff
@@ -33,6 +33,17 @@
       <Version>4.7.0</Version>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
+    <PackageReference Include="Microsoft.Win32.Registry">
```

Comment:
Can you change this to normal PackageReference format? I.e.     `<PackageReference Include="Contoso.Utility.UsefulStuff" Version="3.6.0" />`